book/content/naming/naming.md

```diff
@@ -32,6 +33,32 @@ Let's look at some examples of bad (No) and good (Yes) file names.
 |`JW7d^(2sl@deletethisandyourcareerisoverWx2*.txt` | `1986-01-28_raw-data-from-challenger-o-rings.txt` |
 
 
+### Machine readable
+
+Names of digital components should be easy to understand for computers.
+Computers like names to have no spaces, deliberate use of delimiters, and no special or accented characters.
+Also computers are case sensitive, so for them `cat.txt` and `Cat.txt` are different files.
+
+Good file/folder names are easy to search for (also using regular expressions) and easy to compute on (for example by splitting on `_` characters).
+
+
+### Human readable
+
+To achieve human readability, it is helpful to have a name that contains information on the content of the file/folder or other thing. For URLs this consept is called [clean URL](https://en.wikipedia.org/wiki/Clean_URL).
+
+
+
+### Play well with default ordering
+
+To create a good default ordering, adding number at the beginning of the name, is often a good idea. 
+Imagine you have a folder with als you presentation slides, it would probably be helpful to have them ordered by date. 
+So starting the file names with `year-month-day` (for example `2020-02-21`), does that for you.
+We recommend using something like the [ISO 8601 standard](https://en.wikipedia.org/wiki/ISO_8601) for dates. 
+If you use other number, we recommend left padding them with zeros, because your computer will order `003 < 004 < 020 < 100` as opposed to `100 < 20 < 3 < 4`. 
+From experience we know that naming folders according to a logical number can lead to a mess if the ordering changes.
+For example you have book chapter folders `01_introduction`, `02_naming_files`, `03_naming_folders`, and so on and then you decide to change the order or try to squeeze in a folder in between `01_introduction` and `02_naming_files`, this leads to more downsides than upsides.
```

Comment:
```suggestion
### Optional: Maintain default ordering

To create a good default ordering, i.e sorted in ascending order based on file version or chronology, adding a number or date at the beginning of the name is often a good idea. 
For instance, we often organize all our slide decks created on different dates in the same folder. 
To sort them by date of creation, we can start the file names with `year-month-day` (for example `2020-02-21`).
We recommend using something like the [ISO 8601 standard](https://en.wikipedia.org/wiki/ISO_8601) for dates. 
If you use other number, we recommend left padding them with zeros, because your computer will order `003 < 004 < 020 < 100` as opposed to `100 < 20 < 3 < 4`. 

Naming folders according to a logical number can lead to a mess if the ordering changes in the future.
For example, there is a folder with the book chapters `01_introduction`, `02_naming_files`, and `03_naming_folders`. The author writes a preface of the book and decides to squeeze it before the introduction chapter. This would mean that they will have to rename all the files. 
This happens a lot and clearly, this has more downsides than upsides.
```

Reply:
I fixed a typo in your suggestion. Can we somehow get rid of the "i.e"? 
Otherwise I think it is great!